PKGBUILD

```diff
@@ -2,15 +2,15 @@ pkgdesc="ROS - This package contains generic definitions of geometric shapes and
 url='http://wiki.ros.org/geometric_shapes'
 
 pkgname='ros-noetic-geometric-shapes'
-pkgver='0.7.0'
+pkgver='0.7.2'
 arch=('i686' 'x86_64' 'aarch64' 'armv7h' 'armv6h')
-pkgrel=4
+pkgrel=1
 license=('BSD')
 
 ros_makedepends=(
     ros-noetic-resource-retriever
     ros-noetic-eigen-stl-containers
-    ros-noetic-catkin
+    'ros-noetic-catkin>=0.5.68'
```

Comment:
Typically versioned deps are heavily frowned upon. There is only one package that provides ros-noetic-catkin which we control so there is no need to specify a lowerbound for this.

Reply:
Oh ok, good to know.